charts/k8s-monitoring/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v2
 name: k8s-monitoring
 description: A Helm chart for gathering, scraping, and forwarding Kubernetes telemetry data to a Grafana Stack.
 type: application
-version: 1.1.1
+version: 1.1.2
```

Comment:
Can you change this back? We control the versioning when its time to release"

Reply:
Done at https://github.com/grafana/k8s-monitoring-helm/pull/580/commits/3406dfc64dd8c564986465841cd57a97135899b2